docs/configuration-guides/platforms/aws.md

```diff
@@ -0,0 +1,216 @@
+| `network_mtu`                 | CNI interface MTU (applies only to calico). Use 8981 if using instances types with Jumbo frames.                                                                                           | 1480            | false    |
+| `host_cidr`                   | CIDR IPv4 range to assign to EC2 nodes.                                                                                                                                                    | "10.0.0.0/16"   | false    |
+| `pod_cidr`                    | CIDR IPv4 range to assign Kubernetes pods.                                                                                                                                                 | "10.2.0.0/16"   | false    |
+| `service_cidr`                | CIDR IPv4 range to assign Kubernetes services.                                                                                                                                             | "10.3.0.0/16"   | false    |
+| `cluster_domain_suffix`       | Cluster's DNS domain.                                                                                                                                                                      | "cluster.local" | false    |
+| `enable_reporting`            | Enables usage or analytics reporting to upstream. (applies to Calico only)                                                                                                                 | false           | false    |
+| `certs_validity_period_hours` | Validity of all the certificates in hours.                                                                                                                                                 | 8760            | false    |
+
+### Backend arguments
+
+Default backend is local.
+
+| Argument            | Description                                                  | Default | Required |
+|---------------------|--------------------------------------------------------------|:-------:|:--------:|
+| `local`             | Local backend configuration block.                           | -       | false    |
+| `local.path`        | Location where Lokomotive stores the cluster state.          | -       | false    |
+| `s3`                | AWS S3 backend configuration block.                          | -       | false    |
+| `s3.bucket`         | Name of the S3 bucket where Lokomotive stores cluster state. | -       | true     |
+| `s3.key`            | Path in the S3 bucket to store the cluster state.            | -       | true     |
+| `s3.region`         | AWS Region of the S3 bucket.                                 | -       | false    |
+| `s3.aws_creds_path` | Path to the AWS credentials file.                            | -       | false    |
+| `s3.dynamodb_table` | Name of the DynamoDB table for locking the cluster state.    | -       | false    |
+
+## Installation
```

Comment:
To be consistent with `Uninstalling`

```suggestion
## Installing
```

Reply:
"Installing" or maybe "Creating"/"Destroying"?